### PR TITLE
Release prep for v1.7.3

### PR DESCRIPTION
**Pull request by angelapwen**
Release prep. Bumps some node versions to v16.14.2 to match VSCode's node version.

## Checklist

- [x] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [x] Issues have been created for any UI or other user-facing changes made by this pull request.
- [N/A] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: '16.14.0'
+          : '16.14.0'
```
Comment (aeisenberg) [inline .github/workflows/main.yml:85]: typo?

File: extensions/ql-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
```
Comment (aeisenberg) [inline extensions/ql-vscode/CHANGELOG.md:3]: Can you delete this line?

File: .github/workflows/main.yml (hunk 1 above)
Comment (angelapwen) [reply .github/workflows/main.yml]: Yep! Fixed!

File: extensions/ql-vscode/CHANGELOG.md (hunk 2 above)
Comment (angelapwen) [reply extensions/ql-vscode/CHANGELOG.md:3]: Yep — I forgot whether the automation was smart enough to add it or not.

Comment (angelapwen) [reply extensions/ql-vscode/CHANGELOG.md]: Deleted!

**Review by aeisenberg — APPROVED** [review]

**Pull request merged**